_posts/2021-12-27-trino-2021-a-year-of-growth.md

```diff
@@ -0,0 +1,346 @@
+---
+layout: post
+title:  "Trino 2021 Wrapped: A Year of Growth"
+author: Brian Olsen
+excerpt_separator: <!--more-->
+---
+
+As we reflect on Trino’s 2021 wrapped, one thing that stands out is the 
```

Comment:
As we wrap up the year 2021 in the Trino community, one things stands out. Compared to previous years we have seen even further accelerated, tremendous growth.